pfio/v2/_utils.py

```diff
@@ -0,0 +1,6 @@
+from typing import Any, Dict, Type
+
+
+def format_repr(cls: Type, data: Dict[str, Any]) -> str:
```

Comment:
I want this function included in the tail of `fs.py` to obey the no-util principle.

Reply:
Thank you for the commend. Fixed: 314682840a26a88c69c1f8f208473367772c6320